src/App.jsx

```diff
@@ -13,46 +15,97 @@ export const goodsFromServer = [
   'Jam',
   'Garlic',
 ];
+const SORT_BY_ALPHABET = 'alphabet';
+const SORT_BY_LENGTH = 'length';
```

Comment:
```suggestion
const SORT_TYPES = {
  alphabet: 'alphabet',
  length: 'length',
};
```
If constants are related you can store them in the object